pop_cipher.py

```diff
@@ -5,21 +5,40 @@
 import spotipy
 from spotipy.oauth2 import SpotifyClientCredentials
 import sys
+from sys import stderr
 
-def generate_cipher(input_text, songs):
+MAX_SONG_SEARCH_ATTEMPTS = 5
```

Comment:
this is good. I think eventually it should change to do a validation at the beginning to see if `input_text` can be ciphered given the artists in the json file, if not throw an error.